### ISSUE TITLE
Unchecking "Include in Tab Order" doesn't work as expected

### ISSUE THREAD

**Issue by nlotz63** [mathjax/MathJax#2786]
In the accessibility menu, unchecking "Include in Tab Order"  only works the first time you uncheck it. Although the setting is saved to local storage, returning to a page with math the setting is ignored. In other words, math will be in the tab order.

MathJax version 3.2.0
Windows 10
MS Edge, FireFox, Chrome

To reproduce using the MathJax samples page, go the [MathJax samples page](https://www.mathjax.org/#samples):

1. Tab to some math.
2. Right click and uncheck "include in tab order" (if you previously unchecked the option, you'll see that the option is unchecked, but the setting was ignored.)
3. Confirm math is no longer included in the tab order
4. Refresh the window
5. The expected behavior is tabbing will skip math; however, tab to some math (include in tab order should still be unchecked because of saved preferences)
6. Confirm that the "include in tab order" option is unchecked.
7. Check the "include and tab order" option and then uncheck it. Math should no longer be included in the tab order. 

To summarize, after refreshing the window, it remembers your preference (looking in local storage confirms this); however, it ignores the menu setting and includes math in the tab order. I've tried other settings like explorer and click to zoom and they work as expected.

**Comment by dpvc** [comment]
Sorry for the long delay in getting back to you on this.  We are putting together a new release, and I've made a PR that should resolve the issue.  It turns out that a number of the settings weren't sticky, and this is one of them.  Thanks for the report.
